addon-test-support/@ember/test-helpers/settled.ts

```diff
@@ -174,15 +176,20 @@ export interface SettledState {
   @returns {Object} object with properties for each of the metrics used to determine settledness
 */
 export function getSettledState(): SettledState {
```

Comment:
The documentation for this method needs to be updated to include the new property, and probably cross link to the DebugInfo docs.

Reply:
Roger.